source/_docs/eu-region.md

```diff
@@ -9,59 +9,119 @@ earlyaccess: true
 
-Once granted early access, our Support team will create the new site for you.
+## Region Availability
+Once enabled, this organization feature allows <a href="/docs/change-management/#organizations-roles-and-permissions" data-proofer-ignore>privileged users</a> to designate one of two available regions at the time of site creation:
 
-## Use Terminus to Display Sites and Regions
+* US (Default)
+* EU
 
-In the following examples, replace or assign `$SITE` with your site name or UUID.
+Site resources, including the database container, are provisioned in the specified region. For EU sites, traffic is served through the Global CDN, with the Amsterdam point of presence as origin shield.
 
-To display a specific site's plan and region information:
+
+## Go Live in the EU
+### Create a New Site (Required)
+
+1. Install and authenticate [Terminus](/docs/terminus). If you're already running Terminus, be sure and update to the [latest version](/docs/terminus/updates/) (2.0 minimum required).
+2. Use Terminus to create a new site associated with your organization and include the `--region=eu` option.  
+
+ For example (replace `my-eu-site-name`, `My EU Site Name`, `WordPress` and `My Organization Name` accordingly):
+
+  ```bash
+  terminus site:create my-eu-site-name "My EU Site Name" "WordPress" --org "My Organization Name" --region eu
+  ```
+
+  ![terminus site:create my-eu-site "My EU Site" "WordPress" --org "Rachel Pantheor" --region eu](/source/docs/assets/images/create-site-eu.png)
+
+### Review Site Region (Optional)
+You can view the region from within the Site Dashboard by clicking **Settings** > **About Site** then reviewing **Region**:
+
+![Site Dashboard > Settings > About Site > Region: European Union](/source/docs/assets/images/settings-about-site-region-eu.png)
+
+You can also get this information via Terminus.
+
+<div class="alert alert-info" role="alert">
+<h4 class="info">Note</h4>
+<p markdown="1">Replace or assign `$SITE` with your site name or UUID.</p>
+</div>
+
+#### Display information for a specific site:
 
 <div class="copy-snippet">
   <button class="btn btn-default btn-clippy" data-clipboard-target="#terminus-site-info">Copy</button>
   <figure><pre id="terminus-site-info"><code class="command bash" data-lang="bash">terminus site:info $SITE</code></pre></figure>
 </div>
 
-For a list of sites along with their region:
+#### Display a list of organization sites and their region:
 
 <div class="copy-snippet">
   <button class="btn btn-default btn-clippy" data-clipboard-target="#terminus-site-list">Copy</button>
-  <figure><pre id="terminus-site-list"><code class="command bash" data-lang="bash">terminus site:list</code></pre></figure>
+  <figure><pre id="terminus-site-list"><code class="command bash" data-lang="bash">terminus site:list --org "My Organization Name" --fields name,region</code></pre></figure>
 </div>
 
-## Open Ticket to Prepare to Go Live
+#### Show Relevant Response Header
```

Comment:
something like this for section heading?
Verify Domains Route Correctly
Verify Domains Route through Correct Origin Shield
Veridy Domain Routes through Amsterdam